fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/FlatMapPipelinedCursor.java

```diff
@@ -56,6 +57,13 @@
 @API(API.Status.MAINTAINED)
 @SuppressWarnings("PMD.CloseResource")
 public class FlatMapPipelinedCursor<T, V> implements RecordCursor<V> {
+    private static final CompletableFuture<Boolean> CANCELLED;
```

Comment:
We have this in two places now. Perhaps worth creating `MoreAsyncutil.ALREADY_CANCELLED`